src/community/taskmanager/core/bin/pom.xml

```diff
@@ -135,9 +135,9 @@
     
     <repositories>
       <repository>
-       <id>boundless</id>
-       <name>Boundless Maven Repository</name>
-       <url>https://repo.boundlessgeo.com/main/</url>
+      <id>nexus</id>
```

Comment:
Can we use "osgeo-snapshot" and "osgeo-release" for consistency?

Reply:
I've we're going to have seperate endpoints/repository entries for geotools, "geoserver-releases" and "geoserver-snapshots" might be better - I'll update with those

Reply:
Approach I was expecting:

- geoserver-release and geoserver-snapshot used for publishing
- osgeo-release and osgeo-snapshot groups should be used for dependencies (and contains the geotools, geoserver and possibility GeoSolutions repos).

See [here](https://github.com/geotools/geotools/wiki/Change-from-webdav-to-nexus-repository) for background.

Reply:
All the deployment repository descriptors should share the same name to make auth configuration simpler (in settings.xml) - credentials are keyed to the repository id.
They two groups for dependencies make sense, I'll make that update when we've got GeoTools artifacts